From 328875660a41dc227c42310f536ae64c3a4d4ca8 Mon Sep 17 00:00:00 2001
From: Axel Lin <axel.lin@ingics.com>
Date: Sun, 2 Dec 2012 22:40:24 +0800
Subject: [PATCH 14/14] irqchip: irq-sunxi: Add terminating entry for
 sunxi_irq_dt_ids

The of_device_id table is supposed to be zero-terminated.

Signed-off-by: Axel Lin <axel.lin@ingics.com>
Signed-off-by: Maxime Ripard <maxime.ripard@free-electrons.com>
---
 drivers/irqchip/irq-sunxi.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/irqchip/irq-sunxi.c
+++ b/drivers/irqchip/irq-sunxi.c
@@ -129,7 +129,8 @@ static int __init sunxi_of_init(struct d
 }
 
 static struct of_device_id sunxi_irq_dt_ids[] __initconst = {
-	{ .compatible = "allwinner,sunxi-ic", .data = sunxi_of_init }
+	{ .compatible = "allwinner,sunxi-ic", .data = sunxi_of_init },
+	{ }
 };
 
 void __init sunxi_init_irq(void)
